### PR TITLE
chore(Forms): align docs and add more tests for Iterate.Toolbar and their tools

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Iterate/EditContainer/DoneButton.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function DoneButton(props: Props) {
 
   const doneHandler = useCallback(() => {
     if (hasError) {
-      setShowBoundaryErrors?.(true)
+      setShowBoundaryErrors?.(Date.now())
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Iterate/EditContainer/DoneButton.tsx:35]: Is this correct? 🤔

**Review by langz — APPROVED** [review]

**Review by langz — APPROVED** [review]
👏

File: packages/dnb-eufemia/src/extensions/forms/Iterate/EditContainer/DoneButton.tsx (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/Iterate/EditContainer/DoneButton.tsx]: I revert this ✅

**Pull request merged**